cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -91,8 +95,9 @@ func init() {
 func LoadNetConf(bytes []byte) (*NetConf, logger.Logger, error) {
 	// Default config
 	conf := NetConf{
-		MTU:        "9001",
-		VethPrefix: "eni",
+		MTU:                "9001",
+		VethPrefix:         "eni",
+		PodSGEnforcingMode: sgpp.DefaultEnforcingMode,
```

Comment:
Why do we want SG mode in `NetConf` struct? Is this needed because we no longer can rely on `podVlanID != 0` check now that we have 2 SGPP modes?

Reply:
yes, it's needed since both SGPP mode relies on podVlanID.
the VPC resource controller is agnostic of the SGPP mode, it's purely client-side thing.